.flake8

```diff
@@ -9,4 +9,4 @@ exclude =
     doc/manual/source/conf.py
 max-line-length = 115
 # Ignore some style 'errors' produced while formatting by 'black'
-ignore = W503
+ignore = W503, E231, E271, E272
```

Comment:
Missing whitespace after ":" (E231), multiple spaces before and after keyword (E272 and E271). Errors E272 and E271 were given in docstrings. All errors are ignored for formatting purposes.